### DISCUSSION TITLE
Archive Stacks instead of deleting

### DISCUSSION THREAD

**Discussion by ProtekDev** [louislam/dockge#476] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

I would like to recommend an Archive Feature. 
When the user deletes a stack, the compose.yml could be moved to a different directory, for example, "stack-archive". Additionally, the UI could allow to show/hide the Archive Stacks. 

With such a feature, it would be possible to try some stacks and keep/collect them or create a duplicate/backup.

### ✔️ Solution

- add volume path for stack-archive
- add a ui switch for quickly toggle the visibility of the archive stacks
- when archive stack is started -> move to active stacks
- when active stack gets deleted  -> move to archive stacks
- when deleting the archive stack the actual deletion logic could be used (delete stack entirely)

**Comment by trythatagain** [comment]
Until this happens, you could create a cron job that backs up your compose/stacks base folder using Rsync. It won't take up much room as they are just text files.
